docs/syncer.md

```diff
@@ -11,99 +11,225 @@ users have to install a special component named [syncer](https://github.com/kcp-
+    Make sure to use a KCP kubeconfig for the next steps unless told otherwise.
 
-```sh
-$ kubectl kcp workload sync <mycluster> --syncer-image <image name> -o syncer.yaml
-```
+1. Create an organisation and immediately enter it:
 
-1. Create a kind cluster to back the sync target
+    ```sh
+    $ kubectl kcp workspace create my-org --enter
+    Workspace "my-org" (type root:organization) created. Waiting for it to be ready...
+    Workspace "my-org" (type root:organization) is ready to use.
+    Current workspace is "root:my-org" (type "root:organization").
+    ```
 
-```sh
-$ kind create cluster
-Creating cluster "kind" ...
-<snip>
-Set kubectl context to "kind-kind"
-You can now use your cluster with:
+1. Enable the syncer for a p-cluster:
 
-kubectl cluster-info --context kind-kind
-```
+    ```sh
+    kubectl kcp workload sync <mycluster> --syncer-image <image name> -o syncer.yaml
+    ```
+    Where `<image name>` [one of the syncer images](https://github.com/kcp-dev/kcp/pkgs/container/kcp%2Fsyncer) for your corresponding KCP release (e.g. `ghcr.io/kcp-dev/kcp/syncer:v0.7.5`).
+
+1. Apply the manifest to the p-cluster:
+
+    ```sh
+    $ KUBECONFIG=<pcluster-config> kubectl apply -f syncer.yaml
+    namespace/kcp-syncer-kind-1owee1ci created
+    serviceaccount/kcp-syncer-kind-1owee1ci created
+    secret/kcp-syncer-kind-1owee1ci-token created
+    clusterrole.rbac.authorization.k8s.io/kcp-syncer-kind-1owee1ci created
+    clusterrolebinding.rbac.authorization.k8s.io/kcp-syncer-kind-1owee1ci created
+    secret/kcp-syncer-kind-1owee1ci created
+    deployment.apps/kcp-syncer-kind-1owee1ci created
+    ```
+
+    and it will create a `kcp-syncer` deployment:
+
+    ```sh
+    $ KUBECONFIG=<pcluster-config> kubectl -n kcp-syncer-kind-1owee1ci get deployments
+    NAME     READY   UP-TO-DATE   AVAILABLE   AGE
+    kcp-syncer   1/1     1            1           13m
+    ```
+
+1. Wait for the kcp sync target to go ready:
+
+    ```bash
+    kubectl wait --for=condition=Ready synctarget/<mycluster>
+    ```
+
+### Running a workload
+
+1. Create a deployment:
+
+    ```sh
+    kubectl create deployment kuard --image gcr.io/kuar-demo/kuard-amd64:blue
+    ```
+
+    **Note:** replace "gcr.io/kuar-demo/kuard-amd64:blue" with
+    "gcr.io/kuar-demo/kuard-arm64:blue" in case you're running
+    an Apple M1 based virtual machine.
+
+1. Verify the deployment on the local workspace:
+
+    ```sh
+    $ kubectl rollout status deployment/kuard
+    Waiting for deployment "kuard" rollout to finish: 0 of 1 updated replicas are available...
+    deployment "kuard" successfully rolled out
+    ```
 
 ## For syncer development
 
-Alternately, create a `kind` cluster with a local registry to simplify syncer development by executing the
-following script:
+### Running in a kind cluster with a local registry
+
+You can run the syncer in a kind cluster for development.
+
+1. Create a `kind` cluster with a local registry to simplify syncer development
```

Comment:
Do you need the registry for this? I was able to build the image locally and use `kind load docker-image` to put it on the nodes, as long as you're not using a `:latest` tag it should go straight to storage & not try to pull.